mountpoint-s3/src/cli.rs

```diff
@@ -866,13 +889,13 @@ fn get_region(args_region: Option<String>, instance_info: &InstanceInfo) -> (Str
     (DEFAULT_REGION.to_owned(), false)
 }
 
-fn get_s3_personality(
-    args_personality: Option<S3PersonalityArg>,
+fn infer_s3_personality(
```

Comment:
do we log the personality anywhere? if not, should we?

Reply:
I think we could probably infer it from other log messages, but easy enough to add anyway.